examples/common/pigweed/device_service/device_service.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package chip.rpc;
+
+import 'proto/empty.proto';
+
+// type lengths defined in chip spec 8.2.3.1
+message DeviceInfo {
+  uint32 vendor_id = 1;
+  uint32 product_id = 2;
+  uint32 software_version = 3;
```

Comment:
From different places of spec, vendor ID and product ID are both defined as uint_16, 8.2.3.1 might out of date, I didn't find those type length definition

Reply:
This is just this way to fit within the constraints of proto3, which does not support any native 16-bit integer types.

https://developers.google.com/protocol-buffers/docs/proto3#scalar

Reply:
I see